static/bidder-info/thetradedesk.yaml

```diff
@@ -0,0 +1,21 @@
+endpoint: "https://direct.adsrvr.org/bid/bidder/{{.SupplyId}}"
+maintainer:
+  email: "prebid@thetradedesk.com"
```

Comment:
@andre-gielow-ttd We have sent a verification mail to the above mentioned email address. Please reply to the mail with "Received" message to complete the verification process.

Reply:
Hey @Sonali-More-Xandr I placed the incorrect email address here. Already updated it. Can you re-send the email to Prebid-Maintainers@thetradedesk.com?

Reply:
@Sonali-More-Xandr just following-up on this so I can get this merged.